packages/lesswrong/components/users/UserNameDeleted.tsx

```diff
@@ -10,6 +24,29 @@ const UserNameDeleted = () => {
   </Components.LWTooltip>
 };
 
+const UserNameDeletedWithAdminHover = ({user}: {
+  user: UsersMinimumInfo
+}) => {
+  const {eventHandlers,hover} = useHover();
+  const { LWTooltip } = Components;
+
+  return <span {...eventHandlers}>
+    <LWTooltip
+      title={<div>
+        This user account has been deleted. The username is only visible to site admins, and only visible on hover-over.
+      </div> }
+      inlineBlock={false}
+    >
+      {hover
+        ? <Link to={userGetProfileUrl(user)}>
+            {userGetDisplayName(user)}
+          </Link>
+        : "[anonymous]"
+      }
```

Comment:
I think this hover interaction isn't great - if you hover in the wrong place it keeps flipping between them, which is pretty annoying. Maybe a "click to toggle" is better?

Reply:
One reason for not having click-to-toggle is that the alt-accounts investigation tool is already using click to toggle, so the resulting experience would be two clicks. I think the ideal would be click to reveal, but have it guarantee the width doesn't get narrower when it reveals.